config/config.go

```diff
@@ -53,6 +54,7 @@ type Director struct {
 type Configuration struct {
 	Context            string
 	Certificate        Certificate
+	DbAddress          string
```

Comment:
```suggestion
	RedisDBAddress          string
```

I would suggest to be more explicit of the type of the DB, assuming that we might have any other DB in future.